elastic/kibana
# [Console] Make doc links lead to API Reference in serverless

## Summary

This PR makes the autocomplete endpoints be assigned the [API Reference link](https://www.elastic.co/guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/api-reference.html) for documentation when in serverless.

How to test:
1. Run Kibana in serverless and verify that clicking "Open documentation" for different endpoints leads to [API Reference doc page](https://www.elastic.co/guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/api-reference.html).
2. Run stateful Kibana and verify that each endpoint has its own doc page (as before).



https://github.com/user-attachments/assets/2a55f767-db52-4274-aacb-c8091924cae8
